daemon/authorization/get_album_song.py

```diff
@@ -0,0 +1,43 @@
+import urllib.request
+import json
+
+class Get_album_song:
+    
+    
+
+    def __init__(self,access_token):
+        """
+            Get Spotify catalog information for a single album.
+        """
+        self.headers = {
+            "Authorization": "Bearer " + access_token
+        } 
+# }
+    def get_album_song(self,album_id):
+        """
+            Parameters:
+        """
+        self.album_song_url = "https://api.spotify.com/v1/albums/" + album_id
+        req = urllib.request.Request(url = self.album_song_url,headers=self.headers)
+        page = urllib.request.urlopen(req)
+        contentBytes = page.read()
+        contentBytes = contentBytes.decode("utf-8")
+        self.album_song_json = []
+        d1 =  json.loads(contentBytes )
+
+        for i,t in enumerate(d1["tracks"]["items"]):
+            json_value = {"name :" : t["name"],
```

Comment:
Why did it append ` :` in each key on the dictionary type of `json_value`? 
- ex. `"name :"`

Does it for some data processing?